src/components/App.js

```diff
@@ -1,12 +1,25 @@
-import React from 'react'
-import '../styles/app.css'
+import React, { useState } from "react";
+import "../styles/app.css";
+import Search from "./Search";
+import SearchResults from "./SearchResults"
 
 const App = () => {
+
+  const [searchResults, setSearchResults] = useState([]);
+
   return (
     <div className="app">
-      <p>Hello World!</p>
+      <img
+        className="nasa-logo"
+        src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
+        alt="nasaLogo"
+      />
+      <Search setSearchResults={setSearchResults} />
+      <br />
+      <br />
```

Comment:
Using `<br>` to create space is considered bad practice, as it causes confusion for people using screen readers. If you want to create space between elements, use the `margin-bottom` CSS property on the higher element